scripts/pre_commit_linter.py

```diff
@@ -633,13 +634,41 @@ def _check_bad_patterns(all_files):
     return summary_messages
 
 
+def _check_import_order(all_files):
+    """This function is used to check that each file
+    has imports placed in alphabetical order.
+    """
+    print 'Starting import-order checks'
+    print '----------------------------------------'
+    summary_messages = []
+    all_files = [
+        filename for filename in all_files if not
+        any(fnmatch.fnmatch(filename, pattern) for pattern in EXCLUDED_PATHS)]
+    failed = False
+    for filename in all_files:
+        if isort.SortImports(filename, check=True).incorrectly_sorted:
+            failed = True
+    print ''
+    print '----------------------------------------'
+    print ''
+    if failed:
+        summary_message = '%s   Import order failed' % _MESSAGE_TYPE_FAILED
```

Comment:
This isn't really useful for someone trying to debug and fix a lint error...? You'll need to give more information, including perhaps a list of files (and errors, if possible). 

Also, here and below, the message should be "Import order checks failed"

Reply:
This is just to maintain consistency with the rest of the checks so that import order messages can be incorporated in all messages. The actual error message is printed automatically by Line 649 when SortImports is called.  
Thanks!